src/core/pages/MarkdownMedia.tsx

```
@@ -0,0 +1,73 @@
+  if (!isExternalLink(originalUrl)) {
+    realSrc = buildAssetUrl(originalUrl);
+  }
+
+  return (
+    <img
+      src={realSrc}
+      alt={alt}
+      title={title}
+      {...(params?.width && { width: params.width })}
+      {...(params?.height && { height: params.height })}
+    />
```

Biome (1.9.4)

[error] Provide a text alternative through the alt, aria-label or aria-labelledby attribute

Meaningful alternative text on elements helps users relying on screen readers to understand content's purpose within a page.
If the content is decorative, redundant, or obscured, consider hiding it from assistive technologies with the aria-hidden attribute.

(lint/a11y/useAltText)